test/models/concerns/foreman_git_templates/orchestration/tftp_test.rb

```diff
@@ -62,8 +62,47 @@ class TFTPOrchestrationTest < ActiveSupport::TestCase
     end
   end
 
+  describe '#validate_tftp' do
+    let(:host) { FactoryBot.create(:host, :with_tftp_orchestration, :with_template_url, pxe_loader: 'PXELinux BIOS') }
```

Comment:
let's move `pxe_loader: 'PXELinux BIOS'` to the trait

Reply:
Good idea, but it's actually not related, is it?